source/hardware/controllers/native_instruments_traktor_kontrol_s2_mk3.rst

```diff
@@ -32,108 +32,110 @@ Mixxx mapping
+| **14**. Shift      | Activates secondary functions when pressed                                                                                  |                                                     |
+| button             |                                                                                                                             |                                                     |
++--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
+| **15**. CUE button | CUE default                                                                                                                 | If the CUE point is set, jump to it and stops       |
++--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
+| **16**. Play button| Toggles playing                                                                                                             | Seeks a player to the start and then stops it       |
++--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
+| **17**. Number     | Function depends on current mode                                                                                            |                                                     |
+| buttons            |                                                                                                                             |                                                     |
++--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
+| Hotcue mode        | If hotcue is set, seeks the player to hotcue position. Otherwise set hotcue at current position                             | Clear the hotcue                                    |
++--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
+| Samples mode       | If track is loaded into corresponding slot, go to CUE point and play                                                        | If track is playing, CUE default behaviour.         |
+|                    |                                                                                                                             | Otherwise eject track                               |
```

Comment:
Yes, press without Shift does load the selected track (if none is loaded).
Please add that info here:
```suggestion
| Samples mode       | Load selected track into corresponding slot                                                                                 | If track is playing, CUE default behaviour.         |
|                    | If track is loaded into corresponding slot, go to CUE point and play                                                        | Otherwise eject track                               |
```

Reply:
@olivier-mauras A note: we're now in the final release phase now and would love to get this merged soonish.
If you don't respond or simply don't find time to adopt this change (squash and force-push) in the next let's say two days I'll take the liberty to push this change myself.

Reply:
@ronso0 I'm quite overloaded currently and won't have time to take care of this in the coming weeks, so please go ahead.

Reply:
Okay, thanks for your quick response!